### PR TITLE
fixes #1635: focus the contact form field on open

### LINKED ISSUE #1635: Contact modal text field focus
In contact modals such as the feedback modal, the cursor should initially be placed in the message text field when opening the modal.
Currently, no element in the modal is selected.

### PR DESCRIPTION
**Pull request by FranzSw**
Because `autofocus` is not working with bootstrap modals, I used a jquery listener that focuses the field when the modal is opened. 

fix #1635

### CODE REVIEW

**Review by janno42 — APPROVED** [review]
✔️ Meets requirements
✔️ UI functionality checked

File: evap/static/ts/src/base_template.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// This file contains all typescript code for the base.html template.
+
+// Fix autofocus on bootstrap modals -> Focus the first element with `autofocus` attribute
+document.addEventListener("shown.bs.modal", (e) => {
+  if (!e.target) return;
```
Comment (janno42) [inline evap/static/ts/src/base_template.ts:5]: please use an indentation of 4 spaces

File: evap/evaluation/templates/base.html [hunk 2]
```diff
@@ -70,6 +70,7 @@
 
         <script type="module" src="{% static 'js/csrf-utils.js' %}"></script>
 
+        <script type="module" src="{% static 'js/base_template.js' %}"></script>
```
Comment (niklasmohrin) [inline evap/evaluation/templates/base.html:73]: Please `git mv evap/static/ts/src/base{_,-}template.ts` :)

**Review by niklasmohrin — COMMENTED** [review]
Code looks good!

**Review by He3lixxx — APPROVED** [review]
LGTM if https://github.com/e-valuation/EvaP/pull/1665#discussion_r750506939 is resolved.

**Comment by FranzSw** [comment]
> LGTM if [#1665 (comment)](https://github.com/e-valuation/EvaP/pull/1665#discussion_r750506939) is resolved.

Should be done

**Comment by He3lixxx** [comment]
@FranzSw thanks for contributing!

**Pull request merged**